src/components/header/Header.js

```diff
@@ -0,0 +1,68 @@
+import { useState } from "react";
+import { Link } from "react-router-dom";
+
+import { ReactComponent as Cart } from "../../assets/Cart.svg";
+
+import "./Header.scss";
```

Comment:
There's a vscode extension that renders scss as CSS, would it make sense to use that here so we just use CSS in the project?

Reply:
so basically writing scss but ut converts it to css? Also what is the name of the extension?

Reply:
Live Sass Compiler